advocacy_docs/partner_docs/CohesityDataProtectforPostgreSQL/05-UsingCohesityDataProtectforPostgreSQL.mdx

```diff
@@ -0,0 +1,182 @@
+---
+title: 'Using Cohesity DataProtect for PostgreSQL'
+description: 'Walkthroughs of multiple Cohesity DataProtect for PostgreSQL usage scenarios'
+---
+This section features use cases that show how backups are taken and restored from EDB Postgres Advanced Server using Cohesity DataProtect for PostgreSQL.
+
```

Comment:
The following should all be a heading that is clickable under the "ON THIS PAGE" section just like "Cohesity Recovery of EDB Postgres Advanced Server Data" is
Create a Standard Policy
Protect an EDB Postgres Advanced Server Source with a Policy
Restore an EDB Postgres Advanced Server Instance from a Cohesity Protection Run
EDB Postgres Advanced Server Database Prerequisites Before Recovery Initiation
Cohesity Recovery of EDB Postgres Advanced Server Data